UITests/ClearPrivateDataTests.swift

```diff
@@ -13,7 +13,12 @@ class ClearPrivateDataTests: KIFTestCase, UITextFieldDelegate {
         webRoot = SimplePageServer.start()
     }
 
-    override func tearDown() {
+    override func afterEach() {
+        BrowserUtils.resetToAboutHome(tester())
+    }
+
+
+    override func beforeAll() {
```

Comment:
Oh, OK. Yeah, that's true of all tests -- tests can cause cascading failures since XCTests all use the same single app instance, which sucks.

This change probably isn't terribly useful in the long run since there's no guaranteed order of execution for the tests; if `ClearPrivateDataTests` are followed by anything else, we'll run into the same problem. Rather than a test-specific `beforeAll`, we might want to do this in a base test class as part of [bug 1191476](https://bugzilla.mozilla.org/show_bug.cgi?id=1191476).